src/pages/GameSelectionScreen.vue

```diff
@@ -164,7 +164,7 @@
 import { Component, Vue } from 'vue-property-decorator';
 import Game from '../model/game/Game';
 import GameManager from '../model/game/GameManager';
-import Hero from '../components/Hero.vue';
+import Hero from '../components/v2/Hero.vue';
```

Comment:
Is the intention to create a v2 of all components, or can components that are currently used only in one place be replaced in-place?

Reply:
It's primarily just to map what we have remaining.
Once we have all components inside a v2 folder, we'll migrate them back out. with a big bang replace of `/v2/` to `/`.